<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GreedyProjectionTriangulationParamsDialog</class>
 <widget class="QDialog" name="GreedyProjectionTriangulationParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Greedy Projection Triangulation</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="1">
     <widget class="QSpinBox" name="maxNearestNeighborsSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Maximum nearest neighbors</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Search radius</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="searchRadiusSpinBox">
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Nearest neighbor distance multiplier (Î¼)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="muSpinBox">
      <property name="maximum">
       <double>99.989999999999995</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>GreedyProjectionTriangulationParamsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
